test/demo/embed.html

```diff
@@ -0,0 +1,33 @@
+
```

Comment:
Extra space at the top here.

Also, maybe this demo should be named `iframe-embed.html` and the other `iframe-host.html` or something so it's clear they are a matched set.

Thank you for making these demos!